src/PrestaShopBundle/Form/Admin/Sell/Product/Pricing/ProductSpecificPricePriorityType.php

```diff
@@ -0,0 +1,134 @@
+
+use PrestaShop\PrestaShop\Adapter\Product\SpecificPrice\Repository\SpecificPriceRepository;
+use PrestaShop\PrestaShop\Core\Domain\Product\SpecificPrice\ValueObject\PriorityList;
+use PrestaShopBundle\Form\Admin\Type\TranslatorAwareType;
+use Symfony\Component\Form\Extension\Core\Type\ChoiceType;
+use Symfony\Component\Form\FormBuilderInterface;
+use Symfony\Component\Routing\RouterInterface;
+use Symfony\Component\Translation\TranslatorInterface;
+
+class ProductSpecificPricePriorityType extends TranslatorAwareType
+{
+    /**
+     * @var RouterInterface
+     */
+    private $router;
+
+    /**
+     * @var SpecificPriceRepository
+     */
+    private $specificPriceRepository;
+
+    /**
+     * @param TranslatorInterface $translator
+     * @param array $locales
+     * @param RouterInterface $router
+     * @param SpecificPriceRepository $specificPriceRepository
+     */
+    public function __construct(
+        TranslatorInterface $translator,
+        array $locales,
+        RouterInterface $router,
+        SpecificPriceRepository $specificPriceRepository
+    ) {
+        parent::__construct($translator, $locales);
+        $this->router = $router;
+        $this->specificPriceRepository = $specificPriceRepository;
+    }
+
+    /**
+     * @param FormBuilderInterface $builder
+     * @param array<string, mixed> $options
+     */
+    public function buildForm(FormBuilderInterface $builder, array $options): void
+    {
+        $builder
+            ->add('use_custom_priority', ChoiceType::class, [
+                'choices' => [
+                    $this->buildDefaultPriorityChoiceLabel() => false,
+                    $this->trans('Set a specific order for this product', 'Admin.Catalog.Feature') => true,
+                ],
+                'default_empty_data' => false,
+                'placeholder' => false,
+                'expanded' => true,
+                'multiple' => false,
+                'required' => false,
+                'label' => false,
+                'external_link' => [
+                    'text' => $this->trans('[1]Manage default settings[/1]', 'Admin.Global'),
+                    'href' => $this->router->generate('admin_product_preferences'),
+                    'position' => 'prepend',
+                ],
+            ])
+            ->add('priorities', SpecificPricePriorityType::class, [
+                'row_attr' => [
+                    'class' => 'specific-price-priority-list',
+                ],
+            ])
+        ;
+    }
+
+    /**
+     * @return string
+     */
+    private function buildDefaultPriorityChoiceLabel(): string
+    {
+        $defaultPriorities = implode(' - ', $this->getTranslatedDefaultPriorities());
+
+        return $this->trans(
+            'Use default order: [1]{priority_list}[/1]',
+            'Admin.Catalog.Feature',
+            ['[1]' => '<strong>', '[/1]' => '</strong>', '{priority_list}' => $defaultPriorities]
```

Comment:
```suggestion
            ['[1]' => '<strong> ', '[/1]' => '</strong>', '{priority_list}' => $defaultPriorities]
```

Reply:
```suggestion
            ['[1]' => '<strong>&nbsp;', '[/1]' => '</strong>', '{priority_list}' => $defaultPriorities]
```

Reply:
Maybe you can try and fix this missing space with one of these solutions It's a bit cheating but it's ok Last alternative will be to add CSS padding on this strong element, but then we can do it later